test/run

```diff
@@ -1,5 +1,4 @@
 #! /bin/bash
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS
-make check
-make po/podman.pot
+make check && make po/podman.pot
```

Comment:
You could also just `set -e` before calling make, it'll fast-fail.

Reply:
of course, thanks for pointing it out! Done